app/Platform/Actions/ResumePlayerSessionAction.php

```diff
@@ -139,6 +152,18 @@ public function execute(
 
         $user->playerSessions()->save($playerSession);
 
+        // Update the player's game_recent_players table entry.
+        GameRecentPlayer::upsert(
+            [
+                'game_id' => $game->id,
+                'user_id' => $user->id,
+                'rich_presence' => $presence,
+                'rich_presence_updated_at' => $timestamp,
+            ],
+            ['game_id', 'user_id'],
+            ['rich_presence', 'rich_presence_updated_at']
+        );
```

Comment:
Should this be wrapped in an `if ($presence)` check (like line 88)?

Reply:
I don't think so, due to the conditional on line 130:
```php
// provide a default presence for the new session if none was provided
if (!$presence) {
    $presence = 'Playing ' . $game->title;
}
```
`$presence` will always be truthy on lines 162-165.